zyte_common_items/items.py

```diff
@@ -507,3 +514,103 @@ class ProductList(Item):
     #:
     #: See also ``canonicalUrl``.
     url: str = attrs.field(converter=url_to_str)
+
+
+@attrs.define(slots=True, kw_only=True)
+class BusinessPlace(Item):
+    """Business place, with properties typically seen on maps or business
+    listings."""
+
+    #: Unique identifier of the place on the website.
+    placeId: Optional[str] = None
+
+    #: The main URL that the place data was extracted from.
+    #:
+    #: The URL of the final response, after any redirects.
+    #:
+    #: In case there is no product data on the page or the page was not reached, the returned "empty"
+    #: item would still contain url field and metadata field with dateDownloaded.
+    url: Optional[str] = attrs.field(
+        default=None, converter=attrs.converters.optional(url_to_str), kw_only=True
+    )
+
+    #: The name of the place.
+    name: Optional[str] = None
+
+    #: List of actions that can be performed directly from the URLs on the place page, including URLs.
+    actions: Optional[List[NamedLink]] = None
+
+    #: List of name-value pais of any unmapped additional properties specific to the place.
+    additionalProperties: Optional[List[AdditionalProperty]] = None
+
+    #: The address details of the place.
+    address: Optional[Address] = None
+
+    #: The details of the reservation action,
+    #: e.g. table reservation in case of restaurants
+    #: or room reservation in case of hotels.
+    reservationAction: Optional[NamedLink] = None
+
+    #: List of categories the place belongs to.
+    categories: Optional[List[str]] = None
+
+    #: The description of the place.
+    #:
+    #: Stripped of white spaces.
+    description: Optional[str] = attrs.field(
+        default=None, converter=attrs.converters.optional(lambda x: x.strip())
```

Comment:
I'm not sure it's good to use convertors here; we might want to remove this kind of logic from items, and move it to web-poet processors.

For now, I'd keep only the url_to_str convertors, because they are present in other items/components.